generator/src/test/java/com/scottlogic/deg/generator/walker/RandomReductiveDecisionTreeWalkerTests.java

```diff
@@ -76,12 +77,12 @@ public void shouldProduceTwoRowsOfRandomDataOneRowSpecFromEachIteration() {
     @Test
     public void shouldProduceNoData() {
         when(underlyingWalker.walk(tree)).thenReturn(
-            Stream.of(rowSpec("first-iteration-first-random-row"), rowSpec("first-iteration-second-random-row")),
-            Stream.empty(),
-            Stream.of(rowSpec("third-iteration-first-random-row"), rowSpec("third-iteration-second-random-row"))
+            new DataBagStream(Stream.of(rowSpec("first-iteration-first-random-row"), rowSpec("first-iteration-second-random-row")), false),
```

Comment:
These were already too big, but it would be nice to wrap these around so they don't go much past the 100 char mark.